apps/portals/elportal/src/config/routesConfig.tsx

```diff
@@ -401,13 +401,8 @@ const routes: GenericRoute[] = [
         hideRouteFromNavbar: true,
         synapseConfigArray: [
           {
-            name: 'Markdown',
-            title: 'Contribute Data',
-            props: {
-              ownerId: 'syn27229419',
-              wikiId: '621277',
-              loadingSkeletonRowCount: 15,
-            },
+            name: 'ELContributeYourData',
+            props: undefined,
```

Comment:
This will remove the undesirable side padding
```suggestion
            props: undefined,
            isOutsideContainer: true
```

This boolean determines whether the component will be wrapped in a container. Note that I'm trying to get rid of this with the routing overhaul, so in the future you would simply wrap (or not wrap) the `<ELContributeYourData />` JSX in a `<Container>` element